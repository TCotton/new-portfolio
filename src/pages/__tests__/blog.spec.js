import React from 'react';
import renderer from 'react-test-renderer';
import Blog from '../blog';

describe('Blog', () => {

	let component, instance;

	beforeEach( () => {
		component = renderer.create(
			<Blog />
		);
		instance = component.getInstance();
	});

	it('renders correctly', () => {
		const tree = renderer
			.create(<Blog />)
			.toJSON()
		expect(tree).toMatchSnapshot();
	})

	it('should call shouldComponentUpdate', () => {
		const shouldUpdate = instance.shouldComponentUpdate();
		expect(shouldUpdate).toBe(true);
	});
})
