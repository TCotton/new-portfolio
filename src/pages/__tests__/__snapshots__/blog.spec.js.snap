// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly 1`] = `<div />`;
